docs/tech-reference/yum-plugins.rst

```diff
@@ -627,3 +627,13 @@ Optional Configuration Parameters
 ``force_full``
 Boolean flag to indicate whether or not publish should be done from scratch.
 If unspecified the incremental publish will be performed when possible.
+
+``remove_old_repodata``
+Boolean flag to indicate whether or not repodata passed expiration are removed
+during publish. Files that present in repomd.xml are preserved no matter of 
+their age.
+
+``remove_old_repodata_threshold``
+If ``remove_old_repodata`` is true, this attribute specify maximal age of 
+repodata in seconds that are not removed during publish. Hard coded to 2 weeks.
```

Comment:
Actually this is a default. Maybe it was hard coded before. It's clearly a default so I'm not sure why I asked this be written as hard coded.

s/Hard coded to 2 weeks/This attribute defaults to 14 days/.